composer.json

```diff
@@ -29,7 +30,9 @@
         "phpunit/phpunit": "^7",
         "scrutinizer/ocular": "~1.0",
         "sebastian/phpcpd": "~2.0",
-        "mockery/mockery": "^1.2"
+        "mockery/mockery": "^1.2",
+        "infection/infection": "^0.15.0",
+        "ext-json": "*"
```

Comment:
This dependency is added both as prod and dev.

Reply:
@jose-oliveira Mockery has been there for a while, I didn't add it.

Reply:
I meant the ext-json one

Reply:
Fixed